src/api/folderApi.ts

```diff
@@ -237,12 +237,20 @@ export async function copySharedFolder(
   return await resolveJson(response);
 }
 
-export async function saveSharedFolder({ folderId }: GQLMutationCopySharedFolderArgs, context: Context) {
+export async function saveSharedFolder(
+  { folderId }: GQLMutationCopySharedFolderArgs,
+  context: Context,
+): Promise<string> {
   const response = await fetch(`/myndla-api/v1/folders/shared/${folderId}/save`, context, { method: "POST" });
-  return await resolveJson(response);
+  await resolveNothingFromStatus(response);
+  return folderId;
 }
 
-export async function deleteSharedFolder({ folderId }: GQLMutationCopySharedFolderArgs, context: Context) {
+export async function deleteSharedFolder(
+  { folderId }: GQLMutationCopySharedFolderArgs,
+  context: Context,
+): Promise<string> {
```

Comment:
Dette er forresten ganske misvisende navn. For du kopierer jo ikkje ei mappe, men "favorittmarkerer" den. Og du sletter jo heller ikkje mappa med kun lenka til den.